controllers/photoController.js

```diff
@@ -2,8 +2,76 @@ const url = require("url");
 
 const Photo = require("../models/Photo");
 const Point = require("../models/Point");
+const Map = require("../models/Map");
+
 const { ERROR_MESSAGE } = require("../constants");
 
+const uploadPhoto = async (req, res, next) => {
+  const photo = JSON.parse(req.body.photo);
+  const { createdAt, createdBy } = photo;
+  const { description } = req.body;
+
+  const point = JSON.parse(req.body.point);
+  const { latitude, longitude, placeName } = point;
+
+  const { map } = req.body;
```

Comment:
이렇게 표현할 수도 있을 것 같아요!ㅎㅎ

```suggestion
  const { photo, point, description, map } = req.body;
  const { createdAt, createdBy } = JSON.parse(photo);
  const { latitude, longitude, placeName } = JSON.parse(point);

```

Reply:
성훈님처럼 하는 것도 좋네요-